### PR TITLE
Conversation boxes height fixed

### PR DESCRIPTION
**Pull request by AaskaShah**
Please review and suggest changes.

### CODE REVIEW

File: PowerUp/app/src/main/res/layout/pgame.xml [hunk 1]
```diff
@@ -48,13 +70,7 @@
...
         android:background="@drawable/replay" />
 
-    <ImageView
-        android:id="@id/askerImageView"
-        android:layout_width="@dimen/asker_image_view_width"
-        android:layout_height="@dimen/asker_image_view_height"
-        android:layout_alignParentRight="true"
-        android:layout_below="@id/scenarioNameEditText"
-        android:src="@drawable/marco" />
+
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout/pgame.xml]: Remove this empty line

File: PowerUp/app/src/main/res/layout/pgame.xml [hunk 2]
```diff
@@ -22,15 +22,37 @@
...
+    <ImageView
+        android:id="@+id/askerImageView"
+        android:layout_width="@dimen/asker_image_view_width"
+        android:layout_height="@dimen/asker_image_view_height"
+        android:layout_alignParentRight="true"
+        android:layout_below="@id/scenarioNameEditText"
+        android:src="@drawable/marco" />
+
+    <LinearLayout
+        android:id="@+id/listviewlayout"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout/pgame.xml]: Use camelCase: `listViewLayout`

File: PowerUp/app/src/main/res/layout/pgame.xml [hunk 3]
```diff
@@ -127,24 +143,33 @@
 
     </RelativeLayout>
 
-    <TextView
-        android:id="@+id/questionView"
+    <LinearLayout
+        android:id="@+id/questionViewlayout"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout/pgame.xml]: Same here: `questionViewLayout`

**Comment by alma-castant** [comment]
I left some comments on some minor issues. The branch has conflicts. Please solve them and update your PR so I can test it. Please either solve the issues BEFORE you merge, so you don't get more than 2 commits (yours and the merging one) or create a new PR with one commit only

**Comment by AaskaShah** [comment]
Done :)

**Comment by alma-castant** [comment]
Thanks for solving the issues. The branch still has conflicts. Could you resolve them? Thanks :)

**Comment by AaskaShah** [comment]
Done :)

**Comment by AaskaShah** [comment]
@alma-castant  If this code is correct,should I do the same for landscape mode too ?

**Comment by alma-castant** [comment]
This is still not working as it is supposed to. Can you please test the dimension values? 

In 5" devices on vertical mode, the right box goes over the area it should be in (it continues behind the bottom bar)
In tablets on horizontal mode boxes remain smallet than they should. Please test on all configurations you are targetting before submitting.
Also, please, stick to one commit only ;)
I understand you are not addressing landscape mode on phones here, so that can be addressed in a separate pull request, as the box location needs to be changed

**Comment by AaskaShah** [comment]
I really apologize for not testing on all possible screen sizes.I did so and found that ,
the sizes from the dimens.xml of sw360dp-port is applied to both 4.7" devices(360dp) and 5.2"(432dp) devices.But the difference in the heights is considerable for listview. So could I make another bucket of sw420dp so that all sizes could be adjusted?

**Pull request closed without merging**